packages/exoflex/src/components/Accordion.tsx

```diff
@@ -151,13 +152,17 @@ function Header<T extends Title>({
 
   return (
     <View style={[styles.titleContainer, titleContainerStyle]}>
-      {!!renderIconLeft && renderIconLeft(animatedValue)}
-      <Text style={[styles.title, titleStyle]}>{content && content.title}</Text>
-      {Object.is(renderIconRight, null)
-        ? null
-        : !!renderIconRight
-        ? renderIconRight(animatedValue)
-        : DefaultIcon}
+      <>
```

Comment:
I've checked out these changes and it seems now <View> will throw error when we use more than one child. Is this supposed to work like that? or is it possible we messed up with something? React native [doc](https://reactnative.dev/docs/view#props) say `View is designed to be nested inside other views and can have 0 to many children of any type.`

Reply:
Might be something from the collapsible package we use which attach a gesture handler to the children, requiring specifically only 1 child. Need investigation as I don't remember the detail here.